indexed_gzip/zran.c

```diff
@@ -2384,3 +2384,450 @@ eof:         return ZRAN_READ_EOF;
 
     return ZRAN_READ_FAIL;
 }
+
+/*
+ * Store checkpoint information from index to file fd. File should be opened in
+ * binary write mode.
+ */
+int zran_export_index(zran_index_t *index,
+                      FILE *fd) {
+
+    /*
+     * TODO: Endianness check for fwrite calls. Prefer little-endian to be
```

Comment:
I wouldn't be too worried about endianness-independence at this point in time. Leave the `TODO` comment in though :)